code/game/machinery/doors/windowdoor.dm

```diff
@@ -72,25 +73,25 @@
 		close()
 
 /obj/machinery/door/window/Bumped(atom/movable/AM)
-	if( operating || !density )
+	if(operating || !density)
 		return
-	if (!( ismob(AM) ))
+	if(!(ismob(AM)))
 		if(ismecha(AM))
 			var/obj/mecha/mecha = AM
 			if(mecha.occupant && allowed(mecha.occupant))
 				open_and_close()
 			else
 				do_animate("deny")
 		return
-	if (!( SSticker ))
+	if(!(SSticker))
```

Comment:
unnecessary parenthesis again. also SSticker can't be null lmao, delete this tomfoolery